frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -86,6 +88,10 @@ export const FeatureStrategyMenu = ({
                 variant={variant}
                 size={size}
                 sx={{ minWidth: matchWidth ? '282px' : 'auto' }}
+                disabled={Boolean(disableReason)}
+                tooltipProps={{
+                    title: disableReason ? disableReason : undefined,
+                }}
```

Comment:
I don't think using the disable attribute is right here because it makes the buttons "invisible" to screen readers. As such, unless you can see it and navigate to the button with a mouse, you'll never understand why the button is disabled.

We should instead use the `aria-disabled` attribute, add the correct styling, and set onClick to a no-op function. That way, you don't lose out on anything if you can't mouse over it.